### PR TITLE
Delete timestamp

### PR DESCRIPTION
**Pull request by sdobbert**
Deletes old (-49 hours) granule from image mosaic as part of the dataset-rotation worker.

### CODE REVIEW

File: src/dataset-rotation/README.md [hunk 1]
```diff
@@ -31,4 +31,28 @@ Manages rotation mechanism of incoming datasets.
...
+```
+### Developement
+
+#### VS Code Debugger config 
+
+```json
+{
+    // Verwendet IntelliSense zum Ermitteln möglicher Attribute.
+    // Zeigen Sie auf vorhandene Attribute, um die zugehörigen Beschreibungen anzuzeigen.
+    // Weitere Informationen finden Sie unter https://go.microsoft.com/fwlink/?linkid=830387
```
Comment (chrismayer) [inline src/dataset-rotation/README.md:43]: Comments are not allowed in JSON and being strictly this is no valid JSON. Maybe we better remove the comment. Also it is in German...

File: src/dataset-rotation/index.js [hunk 2]
```diff
@@ -9,60 +10,89 @@ import utc from 'dayjs/plugin/utc.js';
...
+      }
+      // Check if coverage store exists
+      const covStoreObject = await grc.datastores.getCoverageStore(ws, covStore);
 
-    logger.info(`Dataset rotation finished.`);
-    logger.debug({ outputPath: outputPath });
+      if (!covStoreObject) {
+        throw `Coverage store ${covStore} does not exist.`;
+      } else {
+        newPath = await grc.imagemosaics.deleteSingleGranule(ws, covStore, coverage, covFileLocation);
```
Comment (chrismayer) [inline src/dataset-rotation/index.js:76]: `newPath` is not declared as variable, please add `const` or `let`.

**Review by chrismayer — CHANGES_REQUESTED** [review]
Thanks for this.

I have a few comments / suggestions to be looked at before merging this.

One general hint: There are a lot of whitespace changes at unrelated code parts, which makes it hard to review the essential changes of this PR.

**Review by chrismayer — APPROVED** [review]
Code looks good to me, thanks @sdobbert. Untested.

**Comment by chrismayer** [comment]
Tests are failing, with the message

```
/home/runner/work/rabbitmq-worker/rabbitmq-worker/src/dataset-rotation/index.js
Error:   78:8  error  Unnecessary semicolon  no-extra-semi
```
see https://github.com/klips-project/rabbitmq-worker/actions/runs/5507236560/jobs/10036931263?pr=158

Please make the linter happy before merging ;-)

**Review by hblitza — APPROVED** [review]

**Comment by sdobbert** [comment]
Thanks for the review.

**Pull request merged**